Makefile

```diff
@@ -40,6 +40,12 @@ all: copr-rpms rpms
 local:
 	$(MAKE) RPM_DIST="0.$(shell date '+%s')" all
 
+check:
+	black chroma_core/ chroma_api/ chroma_cli/ tests/
```

Comment:
I always run blank without any dirs, it just works.

Reply:
```
nate@gopher ~/src/integrated-manager-for-lustre>$ black
No Path provided. Nothing to do 😴
```

Reply:
`black --check ./`